Add new blog post for iframes (#240)
shipshapecode/website-nuxt: content/blog/posts/2022-06-13-iframes-in-react.md

            const main = contentWindow.document.body.querySelector("main");

            // Because the login form has a dynamic height, observe any size changes and update the iframe height
            const resizeObserver = new ResizeObserver(entries => {
              entries.forEach(entry => {
                setIframeHeight(entry.contentRect.height);
              });
            });

            resizeObserver.observe(main);

            // When the iframe is hiden (i.e. modal is closed), remove any listeners
            const onVisibilityChange = () => {
              resizeObserver.disconnect();
              contentWindow.addEventListener("visibilitychange", onVisibilityChange);
            };

            // Add listener for when iframe is hiden (i.e. modal is closed)
            contentWindow.addEventListener("visibilitychange", onVisibilityChange);
          }}
        />
      </Modal>
    </div>
  );
};

export default MyComponent;
```

And that's it! We now have a page agnostic login modal that utilizes the app's existing login page. Now this component could be used by any developer, on any page, without outside dependencies.

## Future Updates

> There are a couple features missing here that should be taken into consideration before launching to production. Namely, while the iframe is loading what should be shown to the user and what should happen if the iframe has an error while loading? But luckily, with this setup both of those are pretty trivial and mostly come down to UI/UX. A simple `isLoading` state could be used to show/hide a loader and the iframe's built-in `onError` event could be used to handle any loading issues.

Reviewer: ```suggestion
There are a couple features missing here that should be taken into consideration before releasing to production. Namely, while the iframe is loading what should be shown to the user and what should happen if the iframe has an error while loading? Luckily, with this setup both of these are not too complex and mostly come down to UI/UX. An `isLoading` state could be used to show/hide a loader, and the iframe's built-in `onError` event could be used to handle any loading issues.
```
